src/WasmDis.ts

```diff
@@ -1171,3 +1237,185 @@ export class NameSectionReader {
     return new NameSectionNameResolver(functionNames, this._functionLocalNames);
   }
 }
+
+export class DevToolsNameGenerator {
+  private _done: boolean;
+  private _hasNames: boolean;
+  private _functionImportsCount: number;
+  private _memoryImportsCount: number;
+  private _tableImportsCount: number;
+  private _globalImportsCount: number;
+
+  private _functionNames: string[][];
+  private _functionLocalNames: string[][];
+  private _memoryNames: string[][];
+  private _tableNames: string[][];
+  private _globalNames: string[][];
+
+  constructor() {
+    this._done = false;
+    this._hasNames = false;
+    this._functionImportsCount = 0;
+    this._memoryImportsCount = 0;
+    this._tableImportsCount = 0;
+    this._globalImportsCount = 0;
+
+    this._functionNames = null;
+    this._functionLocalNames = null;
+    this._memoryNames = null;
+    this._tableNames = null;
+    this._globalNames = null;
+  }
+
+  private _generateExportName(field: Uint8Array) : string {
+    const exportName = bytesToString(field).replace(/\s/g, '_');
+    return exportName.replace(/\s/g, '_');
```

Comment:
It looks like you're doing this substitution twice.

Reply:
Oh, thanks for catching this! I'll change it.